pkg/object/validation/error.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package validation
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/cli-utils/pkg/object"
+)
+
+type Error interface {
```

Comment:
It would be useful to explain the reason for this interface. It looks like `verror` is the only implementation, so why do we need the interface?

Reply:
I can probably get rid of the interface. I thought it would be useful, but all it ends up doing is avoiding a `*` when casting.

Technically the Unwrap function is a new Go standard, but they didn't make a public interface for it when they added it to the logger. So it doesn't actually need a parent interface to compose it.